specs/app.spec.js

```diff
@@ -5,18 +5,24 @@ const fs = require('fs'),
     DB_CONFIG = require('../dbconfig.json'),
     sinon = require('sinon');
 
-describe('When the user has csv files in the files folder', function () {
+describe('When the user has some csv or txt files to insert', function () {
     this.timeout(10000);
 
     beforeEach(function () {
-        return ensureTablesRemoved(['t1', 't2', 'tr']);
+        return ensureTablesRemoved([
```

Comment:
There are far too many table names here. You should move these closer to where they are being used. i.e. a test which inserts X and Y should clear X and Y, not Z as well.